From d6e8280cd83a08f1b224fa4745de4e3b6f5baa4c Mon Sep 17 00:00:00 2001
From: Dave Young <dyoung@redhat.com>
Date: Mon, 9 Jul 2012 14:57:11 +0800
Subject: [PATCH] 02caps: do not create /bin/sh link

02caps: do not create /bin/sh link

caps.sh use !/bin/bash explictly, so no need to ln -sf bash /bin/sh

OTOH, 00dash will create the symlink /bin/sh, 99base will create it if
there's no /bin/sh symlink. It looks bad to creat /bin/sh in other modules.

If a script want to use bash as command interpreter it should use !/bin/bash
or !/bin/sh in case dash is not installed.

Signed-off-by: Dave Young <dyoung@redhat.com>
---
 modules.d/02caps/module-setup.sh |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/02caps/module-setup.sh b/modules.d/02caps/module-setup.sh
index 6ecb231..c9d94ee 100755
--- a/modules.d/02caps/module-setup.sh
+++ b/modules.d/02caps/module-setup.sh
@@ -14,6 +14,6 @@ install() {
     inst_hook pre-pivot 00 "$moddir/caps.sh"
     inst $(type -P capsh 2>/dev/null) /usr/sbin/capsh
     # capsh wants bash and we need bash also
-    inst /bin/bash && ln -sf bash "${initdir}/bin/sh"
+    inst /bin/bash
 }
 
